conda/models/match_spec.py

```diff
@@ -13,6 +13,28 @@
 from ..exceptions import CondaValueError
 
 
+class SplitSearch(object):
+    """Implements matching on features or track_features. These strings are actually
+       split by whitespace into sets. We want a string match to be True if it matches
+       any of the elements of this set. However, we also want this to be considered an
+       "exact" match for the purposes of the rest of MatchSpec logic, even though it is
+       possible that there are multiple entries in the set, and only one is matching."""
+    __slots__ = ['exact', 'match']
```

Comment:
Oh good, you used slots. I was about to suggest it before I saw you already did.